mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -70,9 +70,9 @@ class Rock_Conv2DOpBase<string mnemonic, list<Type> inputTypes=[F32, F16, BF16],
                    Rock_GemmFeaturesAttr:$features,
                    OptionalAttr<I32Attr>:$derivedBlockSize,
                    OptionalAttr<I32Attr>:$gridSize,
-                   I32ArrayLength<4>:$padding,
-                   I32ArrayLength<2>:$strides,
-                   I32ArrayLength<2>:$dilations,
+                   I64ArrayLength<4>:$padding,
```

Comment:
Ok, I'm glad you flagged me down. Could you make these `index` please?

Reply:
Or at least consider that - given that `index` is the local `ssize_t`, which seems like the right type for these attributes.